### PR TITLE
Multiple deals

### PR DESCRIPTION
**Pull request by RuvendeGroote**

### CODE REVIEW

File: src/react/components/buttons/FaucetButton.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import "../../../styles/navbar.scss";
+import { CredixButton } from "@components/buttons/CredixButton";
+import React from "react";
+
+interface Props {
+	text: string;
+	className?: string;
+}
+
+export const FaucetButton = (props: Props) => {
```
Comment (BouweCeunen) [inline src/react/components/buttons/FaucetButton.tsx:10]: is this used somewhere?

Comment (RuvendeGroote) [reply src/react/components/buttons/FaucetButton.tsx:10]: yes, on the help page and in the navbar

Comment (BouweCeunen) [reply src/react/components/buttons/FaucetButton.tsx:10]: I checked out the branch and I'm getting `Unused constant FaucetButton`, I think there are 2 of them, one in components and one in components/buttons

File: src/credix.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "version": "0.0.0",
+  "version": "0.1.0",
```
Comment (SushantChandla) [inline src/credix.json]: both files are required?
> `src/credix.ts`

Comment (RuvendeGroote) [reply src/credix.json]: nope! will remove

File: src/types/program.types.ts [hunk 3]
```diff
@@ -14,19 +26,6 @@ export enum DealStatus {
...
-	principalAmountRepaid: BN;
-	interestAmountRepaid: BN;
-	timeToMaturityDays: number;
-	goLiveAt: BN;
-	createdAt: BN;
-	leverageRatio: number;
-	underwriterPerformanceFeePercentage: number;
-};
-
 export type PrincipalRepaymentType = { principal: {} };
```
Comment (SushantChandla) [inline src/types/program.types.ts:29]: Is it possible to get the type for these also from `CredixTypes`

File: src/react/components/layouts/DealLayout.tsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { AppLayout } from "./AppLayout";
+
+interface Props {
+	children?: React.ReactNode;
+}
+
+export const DealLayout = (props: Props) => (
```
Comment (SushantChandla) [inline src/react/components/layouts/DealLayout.tsx:8]: why this component?

File: src/react/components/buttons/FaucetButton.tsx (hunk 1 above)
Comment (RuvendeGroote) [reply src/react/components/buttons/FaucetButton.tsx:10]: aha, yeah. I moved it but I guess I messed something up with revert or restore and it got put back. will remove the wrong one

File: src/types/program.types.ts (hunk 3 above)
Comment (RuvendeGroote) [reply src/types/program.types.ts:29]: I have looked into it it doesn't seem to be so. I may be wrong though but I didn't find a way in the source code (no documentation..)

**Review by SushantChandla — APPROVED** [review]

**Review by BouweCeunen — APPROVED** [review]

**Pull request merged**